graphistry/PlotterBase.py

```diff
@@ -1397,8 +1504,8 @@ def server(self, v: Optional[str] = None) -> str:
         """
         from .pygraphistry import PyGraphistry
         if v is not None:
-            PyGraphistry._config['server'] = v
-        return PyGraphistry._config['server']
+            PyGraphistry._session.hostname = v
+        return PyGraphistry._session.hostname
```

Comment:
For backwards compatibility, maybe we create a proxy object for PyGraphistry._config that implements get/set proxies and does depreciation warnings?

Reply:
yes, that is my plan, I wanted to break it first so I found all the places to change

Reply:
(the edits here seem fine, so more a top-level comment)

Reply:
maybe useful:

* gak accesses `PyGraphistry._is_authenticated`, `PyGraphistry._config["api_token"]` - https://github.com/search?q=repo%3Agraphistry%2Fgraph-app-kit%20PyGraphistry&type=code

* louie doesn't really inspect the getters, it mostly manages its own tenants and just calls setters & callers -- `register()`, `org_name()`, `privacy()`, `refresh()`, ...